tests/virt_autotest/login_console.pm

```diff
@@ -116,7 +116,7 @@ sub login_to_console {
 
     if (!get_var('UPGRADE_AFTER_REBOOT')) {
         set_var('REBOOT_AFTER_UPGRADE', '') if (get_var('REBOOT_AFTER_UPGRADE'));
-        if (is_xen_host) {
+        if (is_xen_host && !check_var('START_DIRECTLY_AFTER_TEST', 'qam_virt_install_host_xen')) {
```

Comment:
We have not decide how to carry the default boot yet... kvm host need default boot as well. it will save boot time as well... so how about we make the name of setting more general, such as DEFAULT_BOOT_IS_SET ?

Reply:
If I remember correctly, kvm host boot does not need to be changed -- the default one is the kvm one if kvm is installed.

Reply:
Yes, but it waits for some time in grub menu. Needle matching takes time as well. To support default boot on kvm does not need extra effort if we are going to support xen.

Anyway, it is a minor issue, we can skip it.

Reply:
Yes, but it waits for some time in grub menu. Needle matching takes time as well. To support default boot on kvm does not need extra effort if we are going to support xen.

Anyway, it is a minor issue, we can skip it.